src/components/auth-form/form-password-input/FormPasswordInput.module.css

```diff
@@ -0,0 +1,12 @@
+.PasswordDiv {
+  border: medium solid var(--COLOR-NAVBAR-BACKGROUND);
+  color: var(--COLOR-TEXT);
+}
+
+.PasswordDiv input {
+  border: none;
+}
+
+.PasswordDiv button {
+  color: var(--COLOR-TEXT);
+}
```

Comment:
Here you are using camel case so you can make the first letter of className to lowercase.